queries/superhtml/highlights.scm

```diff
@@ -0,0 +1,49 @@
+(erroneous_end_tag_name) @tag.error
```

Comment:
We only highlight _correct_ code; we explicitly do not do any linting.

(Consider if it makes sense to `; inherit` the  `html` or `html_tags` queries instead of duplicating them.)